tests/TestCase/Utility/XmlTest.php

```diff
@@ -88,7 +88,7 @@ public function testBuild()
         $xml = CORE_TESTS . 'Fixture/sample.xml';
         $obj = Xml::build($xml);
         $this->assertEquals('tags', $obj->getName());
-        $this->assertEquals(2, count($obj));
+        $this->assertCount(2, $obj);
```

Comment:
Sadly this makes the builds fail in [php5.6](https://travis-ci.org/cakephp/cakephp/jobs/309615132#L955).

Reply:
@markstory I've rolled back this.